packages/ui/src/core/autocomplete/autocomplete.props.d.ts

```diff
@@ -13,7 +13,7 @@ export interface AutocompleteProps extends React.HTMLAttributes<HTMLDivElement>
   label?: React.ReactNode;
   hint?: React.ReactNode;
   value: any;
-  options: string[];
+  options: any;
```

Comment:
Instead of any we can have it support the two options we have
```suggestion
type AutocompleteObjectOptionstype= {
  id: number;
  label: string;
 }[];
```
```suggestion
string[] | AutocompleteObjectOptionstype
```